[17.0][IMP] mail_gateway: Simplify button label from 'Gateway message' to 'Gateway'

carlos-lopez-tecnativa opened the pull request:
The purpose is to reduce spacing so that all icons and buttons fit within the window without having to scroll.

Before

<img width="671" height="92" alt="image" src="https://github.com/user-attachments/assets/c3723036-bb80-4fe7-a85f-e6ef1ea6346f" />

After(in combination with this PR https://github.com/OCA/web/pull/3321)
<img width="671" height="92" alt="image" src="https://github.com/user-attachments/assets/6661ee06-14c2-49b9-bc29-f37a43a50f2a" />

TT57968
@Tecnativa @pedrobaeza @pilarvargas-tecnativa @CarlosRoca13 could you please review this?

pedrobaeza reviewed (commented):
Please retranslate at least the term in Spanish in the commit.

carlos-lopez-tecnativa commented:
> Please retranslate at least the term in Spanish in the commit.

Done
<img width="671" height="92" alt="image" src="https://github.com/user-attachments/assets/50715481-95a8-44e4-9904-4c474dc02279" />

pedrobaeza commented:
You have exported from your local installation, and a lot of terms have been added. Please simply edit the original .po to change the term instead.

carlos-lopez-tecnativa committed:
[IMP] mail_gateway: Simplify button label from 'Gateway message' to 'Gateway'

The purpose is to reduce spacing so that all icons and buttons fit within the window without having to scroll.

carlos-lopez-tecnativa commented:
> You have exported from your local installation, and a lot of terms have been added. Please simply edit the original .po to change the term instead.

Done

pedrobaeza commented on mail_gateway/i18n/mail_gateway.pot:
```diff
@@ -241,6 +241,7 @@ msgstr ""
 
 #. module: mail_gateway
 #. odoo-javascript
+#: code:addons/mail_gateway/static/src/components/chatter/chatter.xml:0
```
These changes are unrelated.

pedrobaeza commented on mail_gateway/i18n/mail_gateway.pot:
```diff
@@ -331,7 +332,6 @@ msgstr ""
-#: code:addons/mail_gateway/static/src/components/chatter/chatter.xml:0
 #: code:addons/mail_gateway/static/src/components/composer/composer.esm.js:0
 #, python-format
 msgid "Gateway message"
```
This hasn't been changed though

pedrobaeza reviewed (approved):
Sorry, it's my fault that I haven't checked correctly the latest changes. It seems the OCA bot is not exporting POT files properly. Let's fast-track this for avoiding conflicts with Weblate:

/ocabot merge patch
